scopes/preview/ui/component-preview/compute-preview-scale.ts

```diff
@@ -0,0 +1,7 @@
+
+export function computePreviewScale(width: number, containerWidth: number) {
+  const scale = (containerWidth / width);
```

Comment:
what about height?
if the component is longer and not wider, it might cut the preview incorrectly